pyvista/core/cell.py

```diff
@@ -476,10 +477,15 @@ class CellArray(_vtk.vtkCellArray):
 
     Examples
     --------
-    Create a cell array containing two triangles.
-
+    Create a cell array containing two triangles from the traditional interleaved format
     >>> from pyvista.core.cell import CellArray
     >>> cellarr = CellArray([3, 0, 1, 2, 3, 3, 4, 5])
+
+    Create a cell array containing two triangles from separate offsets and connectivity arrays
```

Comment:
```suggestion
    Create a cell array containing two triangles from separate offsets and connectivity arrays

```

Reply:
ohhh I see. Is there a way to see the rendered docs from the CI build or did you have to build locally?

Reply:
I built it locally to be sure, but you can download successful doc build artifacts via the "checks" tab on the PR -> "build documentation" step in the bottom left -> "docs-build" https://github.com/pyvista/pyvista/suites/14167884129/artifacts/794084378
Come to think of it, I think this artifact is only there when the build is successful...

And FYI you can gather multiple edit suggestions into a single commit with the "add suggestion to batch" button :) https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes